### PR TITLE
add base_url to download_selected

### PR DESCRIPTION
**Pull request by alexandercbooth**
Addresses #1910 by adding base_url to download_selected method

### CODE REVIEW

File: notebook/static/tree/js/notebooklist.js [hunk 1]
```diff
@@ -885,7 +885,7 @@ define([
 
         var item_path = that.selected[0].path;
 
-        window.open(utils.url_path_join('/files', item_path) + '?download=1');
+        window.open(utils.url_path_join(that.base_url, '/files', item_path) + '?download=1');
```
Comment (minrk) [inline notebook/static/tree/js/notebooklist.js]: The `/` should be removed, too, so this should be `, 'files'` instead of `, '/files'`

**Comment by alexandercbooth** [comment]
Thanks for the help, @minrk!

**Comment by minrk** [comment]
Thanks, @alexandercbooth! Welcome to Jupyter 🚀

**Comment by alexandercbooth** [comment]
Thanks, @minrk! I've been using Jupyter for ages and I'm thrilled to start contributing!

**Comment by rgbkrk** [comment]
Yay @alexandercbooth!

**Pull request merged**